.github/workflows/test.yaml

```diff
@@ -14,3 +15,17 @@ jobs:
       run: pip install --requirement=tests/requirements.txt
     - name: Run Tests
       run: pytest --verbose --color=yes tests
+  csl-xml:
+    name: CSL XML Tests
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v2
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: '11'
+    - name: Install dependencies
+      run: sudo apt-get install --yes trang
+    - name: Run Tests
+      run: java -jar trang.jar csl.rnc csl.rng
```

Comment:
With this version, we get the CI error:

```
Run java -jar trang.jar csl.rnc csl.rng
Error: Unable to access jarfile trang.jar
##[error]Process completed with exit code 1.
  Post Checkout Repository
```

That makes sense because `sudo apt-get install --yes trang` is not making `trang.jar` available.

Also it seems that `sudo apt-get install --yes trang` may be adding an additional java version, so it may duplicate efforts with `actions/setup-java@v1`.

@bdarcus do you want to look into this and take it from here? It's just a matter of getting access to the jar file (or an alternative run command) for how we install it.

Reply:
I just edited test.yaml per the below. I'm good if the checks pass?

Reply:
> I just edited test.yaml per the below. I'm good if the checks pass?

Yeah in general. Since this is the first time running it, you might want to look at the logs and see that all the output looks okay.

I don't see any problems on a quick glance.

Reply:
On this:

> Also it seems that sudo apt-get install --yes trang may be adding an additional java version, so it may duplicate efforts with actions/setup-java@v1.

So should we try removing the latter?

Java should be a dependency of trang, so that apt would indeed install it.

Reply:
You could remove the `uses: actions/setup-java@v1` step. It would speed up build times a tiny amount.

The main downside I think to the installation via apt-get, is that you might not get the most recent version of `trang`. Not sure if `trang` is actively updated or whether you want the newest version.

Reply:
> The main downside I think to the installation via apt-get, is that you might not get the most recent version of `trang`. Not sure if `trang` is actively updated or whether you want the newest version.

It's probably fine as is.

It looks like https://github.com/citation-style-language/schema/runs/707588711 uses V20151127, which is two versions behind V20181222 (https://github.com/relaxng/jing-trang/releases). I can't locate a changelog, but RELAX NG itself hasn't changed in decades, and Trang was always robust. The only area we might expect some differences is in support for (embedded) Schematron schema code, I think.